src/providers/auth.provider.tsx

```diff
@@ -96,9 +121,21 @@ export const AuthProvider = ({ children }: { children?: React.ReactNode }) => {
                 login,
                 logout,
                 createAccount,
+                loginWithGoogle
             }}
         >
             {children}
+            <GoogleLogin
+                clientId="587778980690-d0vll2ts3ubnofb21kre5r3qb6959hfo.apps.googleusercontent.com"
+                buttonText="Sign in with Google"
+                onSuccess={(response) => loginWithGoogle(response as GoogleLoginResponseUnion)}
+                cookiePolicy={"single_host_origin"}
+            />
+            <GoogleLogout
+                clientId="587778980690-d0vll2ts3ubnofb21kre5r3qb6959hfo.apps.googleusercontent.com"
+                buttonText="Sign out"
+                onLogoutSuccess={logout}
+            />
```

Comment:
Please remove this. The login button should be in the Login page. The logout should already be handled in the `/me` portal page.